.rubocop.yml

```diff
@@ -5,6 +5,7 @@ AllCops:
   Exclude:
     - 'spec/**/*' # ignore spec tests for now
     - 'vendor/**/*' # ignore non-local files
+    - 'test/**/**/**/*' # ignore serverspec tests
```

Comment:
This is a weird pattern. It ignores everything 3 *or more* levels below `test` (because `**` recursively matches). So it's ignoring everything in and below `test/integration/java_agent/serverspec/` but also everything in and below `test/fixtures/cookbooks/test-helper/`. If the intention is to just ignore serverspec tests, why not `test/**/serverspec/*`? Or why not just ignore `test/**/*`?